src/rt/test/systematic.h

```diff
@@ -219,14 +219,24 @@ namespace Systematic
   private:
     friend class SysLog;
 
-    LocalLog* log;
-
-    ThreadLocalLog() : log(global_logs().acquire()) {}
+    LocalLog* log = nullptr;
+#ifdef USE_FLIGHT_RECORDER
+    ThreadLocalLog()
+    {
+      if constexpr (flight_recorder)
```

Comment:
In what situation can `flight_recorder` be true but `USE_FLIGHT_RECORDER` not be defined?

Reply:
None.  I initially tried to have an constructor with the `constexpr` but that didn't work, so I _changed_ to the `ifdef`, but didn't remove the original try.